deps/Makefile

```diff
@@ -740,6 +732,9 @@ PCRE_OBJ_TARGET = $(build_shlibdir)/libpcre2-8.$(SHLIB_EXT)
 
 # Force optimization for PCRE flags (Issue #11668)
 PCRE_CFLAGS = -O3
+ifneq ($(OS),WINNT)
+PCRE_LDFLAGS = "-Wl,-rpath,'$(build_libdir)'"
```

Comment:
Indeed.  The more I use debian's tools, the more I get annoyed at them, but simultaneously realize they really do know what they're doing when it comes to building software.  :P